### ISSUE TITLE
Repl options errors not shown on UI

### ISSUE THREAD

**Issue by johnjbarton** [google/traceur-compiler#1229]
``` js
// Options: --class=simple
class f {}
```

should show error, but shows nothing
